src/Connector.test.js

```diff
@@ -46,4 +61,27 @@ describe('Connector', () => {
     expect(clientMocks.mockConnect).toHaveBeenCalled();
     expect(error).toBe(errors.connectClient);
   });
+
+  test('listenToCommands: should start listeners on registered devices when there is no error', async () => {
+    const registeredDevices = [mockThing];
+    const client = new Client({ registeredDevices });
+    const connector = new Connector(client);
+    await connector.listenToCommands();
```

Comment:
One side-effect of this operation is that `connector.devices` is filled with the result of `getDevices` call. In this context, we can go beyond the usage of mocks and apply some **stubs** concepts here, i.e. verify the connector's state to validate if that array was filled correctly.

Reply:
I suggest you read [this](https://martinfowler.com/articles/mocksArentStubs.html) article that explains the difference between them.

Reply:
Done! :smiley: